governance.md

```diff
@@ -109,6 +110,12 @@ This is the purpose of Working Groups (WG). The intent is to make
 Working Groups relatively easy to create and to deprecate, once
 inactive.
 
+Working groups do not own any code or subprojects. Instead, they are a place for people to discuss topics that cross SIG boundaries.
+
+Working groups are primarily used to facilitate topics of discussion that are in scope for Kubernetes but that cross SIG lines.
```

Comment:
not sure where to state this, but like a SIG, WG meetings should default to public, be recorded for later viewing, etc.

Reply:
Hmm -- I'll add a note but we should make sure this is handled as part of #1997. SIG stuff is in sig-governance.md.